tests/testthat/test-lancelot-pmcmc.R

```diff
@@ -37,10 +30,80 @@ test_that("can add and remove trajectories from mcstate_pmcmc objects", {
 
 test_that("can compute incidence for a single variable", {
   dat <- reference_data_lancelot_mcmc()
-  cmp <- add_trajectory_incidence(dat$trajectories, c("deaths", "icu"))
-  res <- add_trajectory_incidence(dat$trajectories, "deaths")
-  expect_identical(res$state["deaths_inc", , ],
-                   cmp$state["deaths_inc", , ])
+  cmp <- add_trajectory_incidence(dat$trajectories, c("general", "icu"))
+  res <- add_trajectory_incidence(dat$trajectories, "general")
+  expect_identical(res$state["general_inc", , ],
+                   cmp$state["general_inc", , ])
+})
+
+
+test_that("Can drop predictions from trajectories", {
```

Comment:
Remove this test and the next 4 as they've been deleted from master (they are testing functions that have been deleted)